finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/Sentinels.scala

```diff
@@ -0,0 +1,181 @@
+package com.twitter.finagle.redis.protocol.commands
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.protocol.Commands._
+import com.twitter.finagle.redis.util._
+import com.twitter.finagle.redis.ClientError
+
+import java.net.InetSocketAddress
+
+import org.jboss.netty.buffer.ChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffers
+
+case class SentinelMaster(name: ChannelBuffer)
+  extends Sentinel(SentinelMaster.channelBuffer, Seq(name))
+
+object SentinelMaster extends SentinelHelper {
+  val command = "MASTER"
+  def apply(args: Seq[Array[Byte]]): SentinelMaster = {
+    val name = trimList(args, 2, "SENTINEL MASTER")(0)
```

Comment:
So the main places where this command are used seem to be in stats and tracing.  Do you want them to appear as "SENTINEL" or "MASTER" or "SENTINEL MASTER"?  In Config, it seems that we chose "CONFIG".